src/Analysis/Ast/Test/GenericsTests.cs

```diff
@@ -642,5 +642,20 @@ class LoggedVar(Generic[T]):
             var analysis = await GetAnalysisAsync(code);
             analysis.Should().HaveVariable("x").OfType(BuiltinTypeId.Int);
         }
+
+        [TestMethod, Priority(0)]
```

Comment:
Make separate test class (and file) like for `LintTypeVarTests`, like we have `LintUndefinedVarsTests`

Reply:
Sounds good.